bodhi/server/templates/update.html

```diff
@@ -1033,7 +1033,13 @@ <h3>Related Bugs <span class="badge">${len(update.bugs)}</span></h3>
         % for build in update.builds:
         <tr class="media">
           <td>
-            <a href="https://koji.fedoraproject.org/koji/search?terms=${build.nvr}&type=build&match=glob" target="_blank">
+            <%
+              import urlparse
```

Comment:
I'm not too familiar with mako templates, but I think we can probably move this import statement into the top of the template in another ```<% %>``` block, to more closely resemble PEP-8 Python import style.